### ISSUE TITLE
Secondary Button in Dialog create Axe fails - Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by hegou33** [Sage/carbon#4435]
### Current behaviour

Background color of Dialog causes color contrast issue when secondary button is inside.
![image](https://user-images.githubusercontent.com/38658070/135099915-bcbd5861-e9c2-4797-89cf-302ff6372605.png)

### Expected behaviour

Dialog background color doesn't causes contrast issues.

### CodeSandbox or Storybook URL

https://codesandbox.io/s/elastic-kowalevski-6le0d?file=/src/index.js

### JIRA Ticket (Sage Only)

SBS-20270

### Carbon Version

87.0.1

### What browsers are you seeing the problem on?

Chrome

### What Operating System are you seeing the problem on?

Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by sjohnson-sage** [comment]
Hey @hegou33 - thanks for raising 👍🏻  This is a partial duplicate of https://github.com/Sage/carbon/issues/4417 so I will close it - keep an eye out on that issue for updates
